frontend/context/BalanceProvider.tsx

```diff
@@ -195,6 +198,22 @@ export const BalanceProvider = ({ children }: PropsWithChildren) => {
     () => masterSafeAddress && walletBalances[masterSafeAddress],
     [masterSafeAddress, walletBalances],
   );
+  const agentSafeBalance = useMemo(
+    () =>
+      services?.[0]?.chain_data?.multisig &&
+      walletBalances[services[0].chain_data.multisig],
+    [services, walletBalances],
+  );
+  const isLowBalance = useMemo(() => {
+    if (!safeBalance || !agentSafeBalance) return false;
```

Comment:
they are objects, can be null (undefined actually) only when not loaded

Reply:
think it should just return here if safeBalance || agentSafeBalance don't exist?
otherwise if `isLowBalance == false`  then it means the balance is not low :')